app/src/renderer/components/common/TmBalance.vue

```diff
@@ -16,7 +16,7 @@
       </div>
       <div v-if="rewards" class="top-section">
         <h3>Total Rewards</h3>
-        <h2>{{ rewards }}</h2>
+        <h2>{{ num.short(distribution.totalRewards[bondDenom] || 0) }}</h2>
```

Comment:
we agreed on not using `short` and wait for #2044 
also I think putting this formatting in the computed as it was before is better for testability

Reply:
yeah I missed this one, same as `PageNetwork`